Makefile

```diff
@@ -13,9 +13,14 @@ dev-environment:
 	pip install -e .
 
 docker-build:
-	docker build -t firewatch .
+	docker build -t firewatch . --no-cache
```

Comment:
I think no-cache should be removed from here, there is no point to rebuild all if not needed, docker cache save time

Reply:
True, I was running into issues developing this and added that while debugging. Forgot to remove.